### DISCUSSION TITLE
Pause service blocking

### DISCUSSION THREAD

**Discussion by HunWatt** [AdguardTeam/AdGuardHome#7312] [General]
Hello,
Is it possible to schedule service blocking for a specific client?
For example block facebook, youtube etc. every day 20h-24h on spec. client.

**Comment by schzhn** [comment]
Hello, @HunWatt.
Yes, you can. Use the "Block specific services" and "Pause service blocking" tabs in the pop-up window when creating or editing a client.

**Reply by HunWatt** [reply]
Hello, @schzhn.
I found it, thank you.
